src/app/courses/page.tsx

```diff
@@ -1,11 +1,101 @@
+'use client'
+
+import styled from '@emotion/styled'
 import { COURSES } from '@/data/courses'
+import { FontVariant, Color } from '@/app/theme'
 
 export default function Page() {
   return (
     <main>
-      {COURSES.map(({ title }, i) => (
-        <div key={i}>{title}</div>
-      ))}
+      <h1>Courses</h1>
```

Comment:
Do we have a reset CSS to prevent browsers from applying their own styles? These header tags are often styled differently by browsers